CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# 5.1.0 (2022-02-22)
+
+- Added support for specifying an `aria-label` on the flyout component. [#135](https://github.com/blackbaud/skyux-flyout/pull/135)
+- Added support for the Angular CDK Focus Trap within the flyout component. [#135](https://github.com/blackbaud/skyux-flyout/pull/135)
+- Added support for a `beforeClose` handler to prevent a flyout from closing until parent logic is executed. [#135](https://github.com/blackbaud/skyux-flyout/pull/135)
```

Comment:
```suggestion
- Added support for a `beforeClose` handler to prevent a flyout from closing until the parent logic is executed. [#135](https://github.com/blackbaud/skyux-flyout/pull/135)
```